app/Repositories/MessageRepository.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Gladiator\Repositories;
+
+use Gladiator\Models\Message;
+
+class MessageRepository
+{
+    /**
+     * Create a new message.
+     *
+     * @param  \Gladiator\Models\Contest $contest
+     * @param  array $data
+     * @return \Gladiator\Models\Message
+     */
+    public function create($contest, $data)
+    {
+        $message = new Message;
+        $message->type = $data['type'];
+        $message->key = $data['key'];
+        $message->subject = $data['subject'];
+        $message->body = $data['body'];
```

Comment:
yeah defs! Needed to get it working first cause I had a different architecture before which was grabbing things from separate places and not all coming in as array :) still cleaning a few things up!